scripts/generate_pinval/generate_pinval.py

```diff
@@ -222,6 +227,17 @@ def build_front_matter(
             "avg_price_per_sqft": "${:,.0f}".format(sqft_prices.mean()),
         }
 
+        pred_card_initial_val = _to_num(card_df["pred_card_initial_fmv"])
+
+        pred_card_initial_per_sqft_val = _to_num(
+            card_df.get("pred_card_initial_fmv_per_sqft")
+        )
+
+        if pd.isna(pred_card_initial_per_sqft_val):
+            pred_card_initial_per_sqft_val = pred_card_initial_val / _to_num(
+                card_df["char_bldg_sf"]
+            )
+
```

Comment:
The most annoying part of this code is that we want the price and pred outputs to be formatted like `"$123,456"`, but we need them for summary stat calculations. 

The code here just temporarily transforms them into numeric, while the final front matter figures will remain in the proper format